extension/chrome/settings/modules/compatibility.ts

```diff
@@ -33,11 +34,11 @@ View.run(class CompatibilityView extends View {
     $('pre').text('').css('display', 'block');
     try {
       this.testIndex = 1;
-      const { keys, errs } = await PgpKey.readMany(Buf.fromUtfStr(keyString));
+      const { keys, errs } = await KeyUtil.readMany(Buf.fromUtfStr(keyString));
       for (const err of errs) {
         this.appendResult(`Error parsing input: ${String(err)}`);
       }
-      await this.outputKeyResults(keys);
+      await this.outputKeyResults(await Promise.all(keys.map(key => opgp.key.readArmored(KeyUtil.armor(key)).then(result => result.keys[0]))));
```

Comment:
Todo - check if we have a test for this compatibility page

Reply:
We don't, I'll test it manually + created issue https://github.com/FlowCrypt/flowcrypt-browser/issues/2789

Reply:
Shows an exception when I paste public key from https://flowcrypt.com/pub/tom@flowcrypt.com

```
(17) PK 0 > SK 0 > Verify: [TypeError: Cannot read property 'Symbol(byValue)' of undefined]  
```

Reply:
Good catch. I'll take a look at it.

Reply:
It seems OpenPGP.js changed some APIs quite some time ago. Previously [`Subkey.prototype.verify`](https://openpgpjs.org/openpgpjs/doc/key.js.html#line1085) returned status codes but [currently](https://raw.githubusercontent.com/openpgpjs/openpgpjs/master/dist/openpgp.js) it throws on errors and just returns `undefined` (nothing) on success.

`keyStatus` enum appears to be completely removed from the source code.

Even if it was unchanged I'm still not sure if this code:

```ts
const verifyResNum = await subKey.verify(key.primaryKey);
const veryfyResWord = opgp.enums.read(opgp.enums.keyStatus, 1);
```

shouldn't lookup by the Num result and maybe it should look like this:

```ts
const verifyResNum = await subKey.verify(key.primaryKey);
const veryfyResWord = opgp.enums.read(opgp.enums.keyStatus, verifyResNum);
```

Still, given the API changes this is not needed. I simplified the test. Blame shows unrelated changes due to `openpgp` -> `opgp` renaming and I didn't dig deeper on how and what the second like was inserted...